.github/workflows/turborepo-release.yml

```diff
@@ -320,40 +214,6 @@ jobs:
           mv rust-artifacts/turbo-x86_64-apple-darwin cli/dist-darwin-amd64
           mv rust-artifacts/turbo-x86_64-pc-windows-gnu cli/dist-windows-amd64
 
-      # TODO: This is a duplicate download of artifacts, but since the download path is different,
-      # we will leave this for a future optimization.
-      - name: Download Go artifacts
-        uses: actions/download-artifact@v3
-        with:
-          path: go-artifacts
-
-      - name: Move Go artifacts into place
-        run: |
-          ls go-artifacts
-          mv go-artifacts/turbo-go-cross-${{ needs.stage.outputs.stage-branch }}/turbo_linux_amd64_v1/bin/* cli/dist-linux-amd64
-          chmod a+x cli/dist-linux-amd64/turbo
-          chmod a+x cli/dist-linux-amd64/go-turbo
-          mv go-artifacts/turbo-go-cross-${{ needs.stage.outputs.stage-branch }}/turbo_linux_arm64/bin/* cli/dist-linux-arm64
-          chmod a+x cli/dist-linux-arm64/turbo
-          chmod a+x cli/dist-linux-arm64/go-turbo
-
-          # rust doesn't have a toolchain for arm + windows + gnu, so we just use the exe from the amd64 build
-          # and rely on windows' arm JITer to do the work for us. this is because the go exe cannot be build w/ msvc
-          cp -r go-artifacts/turbo-go-cross-${{ needs.stage.outputs.stage-branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-arm64
-          chmod a+x cli/dist-windows-arm64/turbo.exe
-          chmod a+x cli/dist-windows-arm64/go-turbo.exe
-
-          mv go-artifacts/turbo-go-cross-${{ needs.stage.outputs.stage-branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-amd64
-          chmod a+x cli/dist-windows-amd64/turbo.exe
-          chmod a+x cli/dist-windows-amd64/go-turbo.exe
-
-          mv go-artifacts/turbo-go-darwin-${{ needs.stage.outputs.stage-branch }}/turbo_darwin_amd64_v1/bin/* cli/dist-darwin-amd64
-          chmod a+x cli/dist-darwin-amd64/turbo
-          chmod a+x cli/dist-darwin-amd64/go-turbo
-          mv go-artifacts/turbo-go-darwin-${{ needs.stage.outputs.stage-branch }}/turbo_darwin_arm64/bin/* cli/dist-darwin-arm64
-          chmod a+x cli/dist-darwin-arm64/turbo
-          chmod a+x cli/dist-darwin-arm64/go-turbo
-
       - name: Perform Release
         run: cd cli && make publish-turbo SKIP_PUBLISH=${{ inputs.dry_run && '--skip-publish' || '' }}
```

Comment:
I'm pretty sure this `publish-turbo` target expects `go-turbo` to exists and needs updating. Can you verify that `make publish-turbo SKIP_PUBLISH='--skip-publish'` gets to the point where it would start publishing?

Reply:
It seems to work, at least it fails at the same point as on main